home.css

```diff
@@ -0,0 +1,654 @@
+@font-face {
+  font-family: cocogoose;
+  src: url(fonts/Cocogoose-font.otf);
+}
+
+html,
+body {
+  height: 100%;
+  padding: 0;
+  margin: 0;
+  font-size: 16px;
+  font-family: "lato", sans-serif;
+}
+
+p {
+  line-height: 1.5rem;
+}
+
+.menu-content {
+  display: none;
```

Comment:
- [ ] You are repeating a lot of classes here, so in the spirit of clean code, group all of them, e.g.

```
.menu-content, .top-bar-section, .close-menu, .logo-destop a {
  display: none
}
```